golden_response/staging/query/weather2.json

```diff
@@ -5687,7 +5687,7 @@
           "value": "1943-01"
         },
         {
-          "value": "11.055612903225805"
+          "value": "11.055612903225809"
```

Comment:
This is likely some upstream change in the pipeline we consume wether from that roberts owns.